doc/tutorial/creating-autoinstall-configuration.rst

```diff
@@ -0,0 +1,96 @@
+Creating autoinstall configuration
```

Comment:
I like that this section has it's own page now! I have some nits about improvements I think we could make to this as a tutorial, but I think we could wait for future PRs on this.

Reply:
Could you please file an issue with the suggestions?